crowbar_framework/app/models/batch/export.rb

```diff
@@ -46,7 +46,7 @@ def filename
 
     def persist!
       path.open("w") do |file|
-        file.write content.stringify_keys(true).to_yaml
+        file.write content.stringify_keys.to_yaml
```

Comment:
no, it uses the overloaded version. but the overloaded version which requires (true) somehow didn't work in this scenario. first I have dropped the overloading, but then I saw it it still required by our `crowbar_` bin scripts. but we will hopefully drop them soon, as soon as the `batch build` works in crowbarctl. I guess I will work on that in the near future, and then will continue on this PR. so lets not merge it for now.